src/pages/home/report/ReportActionsView.js

```diff
@@ -130,7 +130,9 @@ class ReportActionsView extends React.Component {
 
         // Only mark as read if the report is open
         if (!this.props.isDrawerOpen) {
-            Report.updateLastReadActionID(this.props.reportID);
+            if (Report.isReportDataUptoDate()) {
+                Report.updateLastReadActionID(this.props.reportID);
+            }
         }
```

Comment:
It might be better to move this check inside `updateLastReadActionID` 
There are other places that could call `updateLastReadActionID` while data is still being fetch

Reply:
Done